src/services/ui/src/components/ExportButton/index.tsx

```diff
@@ -33,6 +35,13 @@ export const ExportButton = <TData extends Record<string, any>>({
     }
 
     for (const item of resolvedData) {
+      const checker = PackageCheck(item as unknown as OsMainSourceItem);
```

Comment:
What's this inline casting for? We should avoid situations where we need to override an inferred type. Maybe `generateExport()` needs its generic updated.